bin/example/vscode/faas-snippets.code-snippets

```diff
@@ -0,0 +1,288 @@
+{
+  "Conversation Util Snippet": {
+    "scope": "javascript,typescript",
+    "prefix": "Conversation Util Snippet",
+    "description": "Adds conversation related methods like fetching conversations or scanning for certain keywords.",
+    "body": [
+      "const { Toolbelt, ConversationContentTypes } = require('lp-faas-toolbelt');",
+      "const conversationUtil = Toolbelt.ConversationUtil();",
+      "// Define parameters",
+      "const conversationId = '${YOUR_CONVERSATION_ID}';",
+      "// Optional Filters",
+      "const contentToRetrieve = [",
+      "  ConversationContentTypes.SDES,",
+      "  ConversationContentTypes.UNAUTH_SDES",
+      "];",
+      "try {",
+      "  // Get conversation",
+      "  const conversation = await conversationUtil.getConversationById(conversationId, contentToRetrieve);",
```

Comment:
You also have to add async to the [exampleFunction](https://github.com/LivePersonInc/faas-cli/blob/master/bin/example/functions/exampleFunction/index.js) to use async/await in the function with the snippets.

```javascript
async function lambda(input, callback) {
  callback(null, `Hello World`);
}
```